### PR TITLE
Updated README.md in DSL with new information

### LINKED ISSUE #1418: dsl: create documentation for next year's group
Create a document and update the README detailing all progress (past and present) in the DSL. This should be in-depth and help others entering the codebase for the first time to get quickly up-to-speed on where the current status is. Additionally, we hope to make a second document indicating the direction next year's group can take the project. This should be a comprehensive list of all the items we hoped to work on/didn't get to, and what might be possible depending on where other teams get in developing their own aspects of the game. 

team members: Andi, William, Hira

### PR DESCRIPTION
**Pull request by baigh1**
This is a pull request for the documentation task. We have made a modification to the README.md that needs to me merged into the main branch. No code implementation.

### CODE REVIEW

File: src/dsl/README.md [hunk 1]
```diff
@@ -85,5 +85,19 @@ python src/parse.py <dsl file> <optional destination> --debug
...
 
 ```
-../../build/chiventure examples/wdl/<filename>.wdl
-```
\ No newline at end of file
+../../build/chiventure examples/wdl/<filename>.wdl 
+```
+
+### Running tests
+The dsl module has built in tests to ensure than any new changes and features implemented do not affect the output of previously implemented features. These tests can be found in the `tests` subdirectory.
```
Comment (alexlee0422) [inline src/dsl/README.md]: "than" should be "that"

**Review by alexlee0422 — APPROVED** [review]
Looks great! The comments are clear and concise and tells what each of the grammars parts are for and what each command does when testing the dsl files. Besides the one typo I saw, this looks great!

File: src/dsl/README.md (hunk 1 above)
Comment (awliu2) [reply src/dsl/README.md]: fixed

**Review by jacknugent1529 — APPROVED** [review]
Looks good to me!

**Comment by jacknugent1529** [comment]
Issue Score: Excellent

Comments: Great work on this minor pull request!

**Pull request merged**